uaclient/entitlements/realtime.py

```diff
@@ -101,3 +108,93 @@ def remove_packages(self) -> None:
                 list(packages),
                 messages.DISABLE_FAILED_TMPL.format(title=self.title),
             )
+
+
+class GenericRealtime(RealtimeKernelEntitlement):
+    variant_name = "generic"
+    title = "Real-time kernel"
+    description = "Generic version of the RT kernel (default)"
+    is_variant = True
+    check_packages_are_installed = True
+
+    @property
+    def incompatible_services(self) -> Tuple[IncompatibleService, ...]:
```

Comment:
For all the variants that are overriding properties of the RealtimeKernelEntitlement like this, I think we'll need to call `super().incompatible_services` and include that in the return value

Reply:
@lucasmoura this comment still applies